docs/pages/bare/using-expo-client.md

```diff
@@ -1,49 +1,69 @@
 ---
-title: Using Expo Go in Bare Workflow
+title: Custom native code in Expo Go
 sidebar_title: Using Expo Go
 ---
 
-import ImageSpotlight from '~/components/plugins/ImageSpotlight'
+import { Terminal } from '~/ui/components/Snippet';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { InlineCode } from '~/components/base/code';
+import SnackInline from '~/components/plugins/SnackInline'
 
-It's not currently possible to include your own native code in Expo Go, so it might surprise you to learn that it is still possible to run your bare project in the Expo Go app.
+The [Expo Go][expo-go] app is capable of running most React Native apps regardless of how they were bootstrapped. Developing any React Native app in Expo Go is useful because you can distribute your project instantly with the members of your [organization](/accounts/account-types/#organizations), no native build or provisioning required.
 
-Inside a freshly initialized bare project, run `npx expo start`, and you can now run it in the client. Read this documentation to learn more about the limitations, why you might want to still use the client despite the limitations, and patterns you can apply to make this work well for you.
+The main drawback of developing in Expo Go is that it is impossible to send custom native code over-the-air to the Expo Go app, meaning you will need to do one of two things:
 
-## What are the limitations?
+1. Create a [development build](/development/introduction) to use custom native code, and distribute with [internal distribution](/build/internal-distribution) or TestFlight.
+2. Conditionally disable unsupported native features and distribute with Expo Go. This guide will demonstrate how achieve this.
 
-You will not be able to use the parts of your app that require custom native code. To run your bare app in Expo Go, you need to avoid calling any custom native code (native code that isn't included in the Expo SDK). For some apps this may mean that you won't be able to use the Expo Go app almost at all &mdash; for example, if your app depends on custom native code for something as fundamental as navigation or state management (eg: Realm or the Firebase native SDK) then not much of your app will be usable in the client. If your app only has some in app purchases, analytics, a custom map view, an AR view, and so on, then this may actually work great for you &mdash; that particular functionality would not be usable in Expo Go but the rest of the app still would be.
+## Usage
 
-## Why might you want to do this?
+Inside any React Native app, start a [development server with Expo CLI](/workflow/expo-cli#develop):
 
-There are a number of benefits to keeping your project runnable in the Expo Go app.
+<Terminal cmd={[
+"$ npx expo start",
+]} cmdCopy="npx expo start" />
```

Comment:
```suggestion
<Terminal cmd={["$ npx expo start"]} />
```

I think this could be a single line since it does not make a difference when represented in the docs.